bitbucket-pipelines.yml

```diff
@@ -0,0 +1,111 @@
+# These are yaml anchors, reused later
+x-bash-env-steps:
+  - &bash_env_export export BASH_ENV="$BITBUCKET_CLONE_DIR/.bashrc"
+  - &bash_env_source source $BASH_ENV
+
+# This is a larger yaml anchor, reused for each pipeline
+default_steps: &default_steps
+  - step:
+      name: Configure Environment Variables
+      services:
+          - docker
+      caches:
+          - docker
+      script:
+        - *bash_env_export
+        - export CI_BRANCH=${BITBUCKET_BRANCH:-master}
+        - export CI_BUILD_NUMBER=$BITBUCKET_BUILD_NUMBER
+        - export CI_PROJECT_USERNAME=$BITBUCKET_REPO_OWNER
+        - export CI_PROJECT_REPONAME=$BITBUCKET_REPO_SLUG
+        - export CI_PROJECT_NAME=$BITBUCKET_REPO_FULL_NAME
+        - /build-tools-ci/scripts/set-environment
+      artifacts:
+        - .bashrc
+  - parallel:
+    - step:
+        name: Static Tests
+        services:
+          - docker
+        caches:
+          - docker
+          - composer
+        script:
+          - *bash_env_export
+          - *bash_env_source
+          - ./.ci/test/static/run
+    - step:
+        name: Build PHP
+        services:
+          - docker
+        caches:
+          - docker
+          - composer
+        script:
+          - *bash_env_export
+          - *bash_env_source
+          - ./.ci/build/php
+        artifacts:
+          - web/**
+          - vendor/**
```

Comment:
Should this include `.gitignore`, `drush/`, and `console/` as well, which may be modified during `build/php`?

Reply:
Yes, once I make a PR to the D8 example repo. They aren’t necessary for WP

Reply:
For `.gitignore` the prepare for Pantheon script is being run in the deploy to Pantheon job, rather than persisting `.gitignore` from the PHP build job.

This will ensure `.gitignore` is correct and there aren't lingering nested `.git` directories, especially if folks add other build steps besides PHP, that won't run the prepare for Pantheon script.